docs/sql-ref-datatypes.md

```diff
@@ -631,4 +631,71 @@ from pyspark.sql.types import *
 </tr>
 </table>
 </div>
+
+<div data-lang="sql"  markdown="1">
+
+The following table shows the type names as well as aliases used in Spark SQL parser for each data type.
+
+<table class="table">
+<tr>
+  <th style="width:50%">Data type</th>
+  <th>SQL name</th></tr>
+<tr>
+  <td> <b>BooleanType</b> </td>
+  <td> boolean </td>
```

Comment:
Shall we capitalize SQL types like `BOOLEAN` and `INT`?

Reply:
Copy, change all the type name. Done in 99d0db6.